features/mfa.feature

```diff
@@ -145,6 +145,32 @@ Feature: Prompt for MFA credentials
       |           |   TOTP   | , backup codes | does not support WebAuthn |   TOTP           |
       |           |          |   backup codes | does not support WebAuthn |      backup code |
 
+
+  Scenario: Defaulting to the manager despite having a used mfa
+    Given I provide credentials that have a manager and a used TOTP
+      And the user's browser supports WebAuthn
+    When I login
+    Then I should see a prompt for a manager rescue code
```

Comment:
I assumed that since that's how it was written originally, that's what we wanted. I think what it is referring to is when a person has already requested a code via their manager.

Reply:
Documenting here what we discussed this directly:
I was mis-interpreting "have/has a manager" to mean "have/has a manager email address for that user" rather than "have/has a manager code because that user already requested one." If the user has already requested that a code be sent to their manager, then in that case we _do_ want to show the manager-code prompt first.